test/core/derivatives/loss_settings.py

```diff
@@ -28,7 +28,7 @@
 example = {
     "module_fn": lambda: torch.nn.CrossEntropyLoss(reduction="mean"),
     "input_fn": lambda: torch.rand(size=(2, 4)),
-    "target_fn": lambda: classification_targets(size=(2,), num_classes=2),
+    "target_fn": lambda: classification_targets(size=(2,), num_classes=4),
```

Comment:
Why does `num_classes` change?

Reply:
Because, the input is (N, C) with batch size N and number of classes C. This correction puts the targets in line with the input.

Reply:
Makes sense.